multicluster/config/overlays/member/kustomization.yaml

```diff
@@ -0,0 +1,31 @@
+namespace: kube-system
+
+namePrefix: antrea-multicluster-
+
+commonLabels:
+  app: antrea
+
+bases:
+  - ../../crd
+  - ../../default
+
+resources:
+  - secret_viewer_role.yaml
+  - secret_viewer_role_binding.yaml
```

Comment:
do we still need this standalone secret view and role binding? I think secret view role has been added via `additional_role_patch.yaml`

Reply:
You are right, its not needed.